.github/workflows/linters.yml

```diff
@@ -45,4 +45,4 @@ jobs:
           npm install --save-dev stylelint@13.3.x stylelint-scss@3.17.x stylelint-config-standard@20.0.x stylelint-csstree-validator
           [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.stylelintrc.json
       - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
+        run: npx stylelint "**/*.scss"
```

Comment:
```suggestion
      run: npx stylelint "**/*.{css,scss}"
```

Reply:
It was mention in the linter config that we do this
+ we are only using sass

Reply:
link https://github.com/microverseinc/linters-config/tree/master/html-css#troubleshooting

Reply:
Hello, I am so sorry for the inconvenience.  
Please submit another review.